### PR TITLE
[Wiki] Overarching review and lingo update

### PR DESCRIPTION
**Pull request by matebarabas**
# Description

This is meant to be an overarching review and update of the wiki documentation and its lingo, in preparation of adding more content.

This is a draft PR, as the work has just begun, but I'd like to be as transparent as possible. 
So far, the home page & the context pages have been reviewed and updated, and all images were consolidated into a folder structure representing the bigger chapters they belong to. Not referenced/used images have been deleted.

# Type of Change

- [x] Update to documentation

# Checklist

- [x] I'm sure there are no other open Pull Requests for the same update/change
- [ ] My corresponding pipelines / checks run clean and green without any errors or warnings
- [ ] My code follows the style guidelines of this project
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation (readme)
- [ ] I did format my code

### CODE REVIEW

File: docs/wiki/Solution creation.md [hunk 1]
```diff
@@ -2,24 +2,30 @@
```
[the remark this answers is not in the document]
Comment (matebarabas) [reply docs/wiki/Solution creation.md:2]: Yep, I figured out already. :) Will do.

**Review by MariusStorhaug — CHANGES_REQUESTED** [review]
First round of feedback

File: docs/wiki/Contribution guide - Contribution flow.md [hunk 2]
```diff
@@ -1,80 +1,77 @@
...
-- [Create or pick up an issue](#Create-or-pick-up-an-issue)
-- [Environment setup](#Environment-setup)
-- [Implement the contribution](#Implement-the-contribution)
-- [Validate the contribution](#Validate-the-contribution)
-- [Open a PR](#Open-a-PR)
+- [Create or pick up an issue](#create-or-pick-up-an-issue)
+- [Environment setup](#environment-setup)
+- [Implement the contribution](#implement-the-contribution)
+- [Validate the contribution](#validate-the-contribution)
+- [Open a PR](#open-a-pr)
```
Comment (MariusStorhaug) [inline docs/wiki/Contribution guide - Contribution flow.md:13]:
```suggestion
- [Create or pick up an issue](#create-or-pick-up-an-issue)
- [Environment setup](#environment-setup)
- [Implement the contribution](#implement-the-contribution)
- [Validate the contribution](#Validate-the-contribution)
- [Open a PR](#Open-a-pr)
```

File: docs/wiki/Home.md [hunk 3]
```diff
@@ -1,74 +1,77 @@
...
-- [Known issues](./Known%20issues)
+- [The context](./The%20context.md)
+  - [CARML overview](./The%20context%20-%20CARML%20overview.md)
+    - [The library](./The%20context%20-%20CARML%20library.md)
+    - [The CI environment](./The%20context%20-%20CARML%20CI%20environment.md)
+- [Getting started](./Getting%20started.md)
+  - [**Scenario 1:** Onboard module library and CI environment](./Getting%20started%20-%20Scenario%201%20Onboard%20module%20library%20and%20CI%20environment.md)
+    - [Check namePrefix availability](./Getting%20started%20-%20Check%20NamePrefix%20availability.md)
+  - [**Scenario 2:** Consume module library only](./Getting%20started%20-%20Scenario%202%20Consume%20library.md)
+  - [**Scenario 3:** Contribute](./Getting%20started%20-%20Scenario%203%20Contribute.md)
```
[the remark this answers is not in the document]
Comment (matebarabas) [reply docs/wiki/Home.md]: Swapped.

File: docs/wiki/The context - CARML CI environment.md [hunk 4]
```diff
@@ -1,36 +1,36 @@
...
 ### _Navigation_
 
 - [Deployment flow](#deployment-flow)
   - [Module versioning](#module-versioning)
-- [Where does the CARML CI environment fit in?](#where-does-this-platform-fit-in)
+- [Where does the CARML CI environment fit in?](#where-does-the-carml-ci-environment-fit-in)
 
 ---
 
 # Deployment flow
```
[the remark this answers is not in the document]
Comment (matebarabas) [reply docs/wiki/The context - CARML CI environment.md:18]: CI environment definition added.

File: docs/wiki/Known issues.md [hunk 5]
```diff
@@ -1,34 +1,39 @@
...
 
 ### _Navigation_
 
 - [Module specific](#module-specific)
+  - [Microsoft.AAD/DomainServices](#microsoftaaddomainservices)
+  - [Microsoft.KubernetesConfiguration/extensions](#microsoftkubernetesconfigurationextensions)
+  - [Microsoft.KubernetesConfiguration/fluxConfigurations](#microsoftkubernetesconfigurationfluxconfigurations)
+  - [Microsoft.Network/vpnGateways](#microsoftnetworkvpngateways)
+  - [Microsoft.Network/virtualHubs](#microsoftnetworkvirtualhubs)
+  - [Microsoft.Network/vpnSites](#microsoftnetworkvpnsites)
```
Comment (eriqua) [inline docs/wiki/Known issues.md:13]: Just a heads-up for pulling latest changes as we just merged an update to this page.

File: docs/wiki/The CI environment - Deployment validation.md [hunk 6]
```diff
@@ -21,17 +21,17 @@ The deployment validation phase can be divided into three steps, running in sequ
...
+The template validation step performs a dry-run with each parameter file in the module's `'.parameters'` folder
 
-In particular, the step executes a `Test-AzDeployment` cmdlet (_the command may vary based on the template schema_) for each provided module parameter file to verify if the template would be able to be deployed using them.
+In particular, the step runs a `Test-AzDeployment` cmdlet (_the command may vary based on the template schema_) for each provided module parameter file to verify if the template could be deployed using them.
 
 The intention of this test is to **fail fast**, before getting to the later deployment step. The template validation could fail either because the template is invalid, or because any of the parameter files is configured incorrectly.
 
 # Azure deployment validation
 
 This step performs the actual Azure deployments using each available & configured module parameter file. The purpose of this step is to prove the module can be deployed in different configurations based on the different parameters provided. Deployments for the different variants happen in parallel.
```
[the remark this answers is not in the document]
Comment (MariusStorhaug) [reply docs/wiki/The CI environment - Deployment validation.md:35]: Agreed to take this feedback to a new issue, and close the point in the PR.

**Pull request merged**